packages/react/rollup.config.js

```diff
@@ -1,17 +1,20 @@
 import babel from "rollup-plugin-babel";
-import commonjs from "rollup-plugin-commonjs";
 
 module.exports = {
     input: "./src/index.js",
     output: {
-        file: "./dist/bundle.js",
-        format: "cjs" 
+        file: "./dist/bundle/bundle.js",
```

Comment:
is there an extra space here? the indenting doesn't look right on github

Reply:
Probably ... I'm guessing the formatting was hit with some weird copypasta